lib/vmpooler/pool_manager.rb

```diff
@@ -705,6 +705,8 @@ def execute!
 
       # Clear out the tasks manager, as we don't know about any tasks at this point
       $redis.set('vmpooler__tasks__clone', 0)
+      # Clear out vmpooler__migrations since stale entries may be left after a restart
+      $redis.del('vmpooler__migration')
```

Comment:
vmpooler__tasks__clone is a counter that has its number increased, and decreased, as clone operations start and finish.

Rather than replicate that model I thought it would be preferred to know which VMs are in the process of being migrated, so I create a set for vmpooler__migration, the vm name is a member of the set, and we count the set members to determine whether we have hit the migration limit. My thought was that it would be helpful to be able to find out which VMs are in the process of being migrated if ever we need to, which I have found useful when validating that an entry is stale.